apps/fpc/fsw/unit_test/fpc_test_utils.cpp

```diff
@@ -313,8 +324,8 @@ double FPC_Test_GetChecksum(FPC_ConfigTbl_t *pTbl)
                + (double)pTbl->FPC_Runway_Takeoff.MAX_PITCH_RADIANS
                + (double)pTbl->FPC_Runway_Takeoff.MAX_ROLL_RADIANS
                + (double)pTbl->FPC_Runway_Takeoff.AIRSPD_SCL
-               + (double)pTbl->FPC_Runway_Takeoff.AIRSPD_MIN
-               + (double)pTbl->FPC_Runway_Takeoff.CLMBOUT_DIFF;
+               + (double)pTbl->FPC_Runway_Takeoff.RWTO_AIRSPD_MIN
+               + (double)pTbl->FPC_Runway_Takeoff.RWTO_CLMBOUT_DIFF;
 
     printf("FPC_Test_GetChecksum:\n");
```

Comment:
What is the purpose of the print and std out statements?  If these are manual inspection points, we need to replace these with automated inspection.  If its just for informational purposes during debugging, this is ok.

Reply:
It's just for informational purposes, due to precision.(std::cout.precision(17))
In either way, the checksum values are the same and the test results are the same.